client/src/Components/CountryBorder/CountryBorder.less

```diff
@@ -2,6 +2,10 @@
   border: 1px solid rgba(0, 0, 0, 0.2);
   box-sizing: border-box;
   background: rgba(100, 99, 99, 0.2);
+  .ScratchCard__Container {
+    border: 3px solid red;
+    cursor: url(coincursor.png), auto;
```

Comment:
Coin looks good, but it looks a little too big to me. Maybe you could make it a little smaller?

Reply:
Reduced size by 25%